src/main/java/com/tomtom/services/configuration/implementation/Configuration.java

```diff
@@ -532,6 +534,40 @@ private static boolean isValidMatchString(@Nonnull final String match) {
     }
 
     @Nonnull
+    private static List<NodeDTO> getChildNodeListFromConfiguration(
```

Comment:
List<NodeDTO> may be replaced with <T> to make it generic and remove duplication.
This means you need to add a parameter with type T to the param list as well.